### DISCUSSION TITLE
Syntax Error When Adding Firefox Profile.

### DISCUSSION THREAD

**Discussion by bapoi555** [TreborNamor/BestbuyBot#59] [Q&A] [answered]
Hello,
It seems that I have a problem w/ the code... After I imputed my Firefox about:profiles Root Directory it started giving me error messages. By the looks of it probably has a problem with the quotations. How can I fix it? Thank you for your help!

 ![image](https://user-images.githubusercontent.com/85934149/122033226-a03a5080-cdd0-11eb-8206-4e97ef686ece.png)

**Comment by TreborNamor** [comment, marked as answer]
oh, I see the problem. You just need to delete line 63 and add your text to this section here highlighted in blue. That is where your profile goes in script. As well, I just updated script a few minutes ago. You should probably use new script instead.

![image](https://user-images.githubusercontent.com/59729477/122041751-e676bc00-cd9e-11eb-9260-05698451f9a6.png)
